cfme/tests/cli/test_appliance_console.py

```diff
@@ -0,0 +1,159 @@
+import pytest
+from wait_for import wait_for
+from utils import version
+from utils.log_validator import LogValidator
+
+
+def test_black_console_set_hostname(request, appliance):
+    hostname = 'test.com'
+    if appliance.version >= "5.8":
+        commands = ['ap\n', '\n', '1\n', '4\n', '{hostname}\n'.format(hostname=hostname)]
```

Comment:
On thinking about this, I think it would be better to
a) name this command_set
b) Make it a tuple so it''s immutable